bulk/text.py

```diff
@@ -1,46 +1,54 @@
-from tornado.ioloop import IOLoop
-
 import numpy as np
 import pandas as pd
-
-from bokeh.io import curdoc
 from bokeh.layouts import column, row
-from bokeh.models import (Button, ColumnDataSource, DataTable, TableColumn, TextInput)
+from bokeh.models import Button, ColumnDataSource, TextInput, DataTable, TableColumn, ColorBar
 from bokeh.plotting import figure
-from bokeh.models import DataTable, TableColumn
 
-def bulk_text(path):
+from .utils import get_color_mapping, get_datatable_columns
 
+
+def bulk_text(path):
     def bkapp(doc):
         df = pd.read_csv(path)
-        saved_idx = []
-        highlighed_idx = []
+        highlighted_idx = []
 
-        source = ColumnDataSource(data=dict())
-        source_orig = ColumnDataSource(data=df)
+        mapper, df = get_color_mapping(df)
+        datatable_columns = get_datatable_columns(df)
+        columns = [
```

Comment:
Because we're adding more columns now, the text truncates in the UI.

<img width="1147" alt="image" src="https://user-images.githubusercontent.com/1019791/174257787-10313618-916d-4406-9fc8-7e8ab7349279.png">

Because we're most interested in the text here, I'm not in favour of adding any columns that might make it harder to inspect it. Can you make sure that we only show the text column here?